CHANGELOG.md

```diff
@@ -1,17 +1,124 @@
+- Addition of the `shopify_timezone` variable, which converts ALL timestamps included in the package (including `_fivetran_synced`) to a single target timezone in IANA Database format, ie "America/Los_Angeles" ([PR #41](https://github.com/fivetran/dbt_shopify_source/pull/41)).
+- The declaration of passthrough variables within your root `dbt_project.yml` has changed (but is backwards compatible). To allow for more flexibility and better tracking of passthrough columns, you will now want to define passthrough columns in the following format ([PR #40](https://github.com/fivetran/dbt_shopify_source/pull/40)):
+> This applies to all passthrough columns within the `dbt_shopify_source` package and not just the `customer_pass_through_columns` example. See the README for which models have passthrough columns.
+```yml
+vars:
+  customer_pass_through_columns:
+    - name: "my_field_to_include" # Required: Name of the field within the source.
+      alias: "field_alias" # Optional: If you wish to alias the field within the staging model.
+      transform_sql: "cast(field_alias as string)" # Optional: If you wish to define the datatype or apply a light transformation.
+```
+- The following *source* fields have been added to (➕) or removed from (➖) their respective models ([PR #39](https://github.com/fivetran/dbt_shopify_source/pull/39), [PR #40](https://github.com/fivetran/dbt_shopify_source/pull/40)):
+  - `shopify__orders`:
+    - ➕ `total_discounts_set`
+    - ➕ `total_line_items_price_set`
+    - ➕ `total_price_usd`
+    - ➕ `total_price_set`
+    - ➕ `total_tax_set`
+    - ➕ `total_tip_received`
+    - ➕ `is_deleted`
+    - ➕ `app_id`
+    - ➕ `checkout_id`
+    - ➕ `client_details_user_agent`
+    - ➕ `customer_locale`
+    - ➕ `order_status_url`
+    - ➕ `presentment_currency`
+    - ➕ `is_confirmed`
+  - `shopify__customers`:
+    - ➕ `note`
+    - ➕ `lifetime_duration`
+    - ➕ `currency`
+    - ➕ `marketing_consent_state` (coalescing of `email_marketing_consent_state` and deprecated `accepts_marketing` field)
+    - ➕ `marketing_opt_in_level` (coalescing of `email_marketing_consent_opt_in_level` and deprecated `marketing_opt_in_level` field)
+    - ➕ `marketing_consent_updated_at` (coalescing of `email_marketing_consent_consent_updated_at` and deprecated `accepts_marketing_updated_at` field)
+    - ➖ `accepts_marketing`/`has_accepted_marketing`
+    - ➖ `accepts_marketing_updated_at`
+    - ➖ `marketing_opt_in_level`
+  - `shopify__order_lines`:
+    - ➕ `pre_tax_price_set`
+    - ➕ `price_set`
+    - ➕ `tax_code`
+    - ➕ `total_discount_set`
+    - ➕ `variant_title`
+    - ➕ `variant_inventory_management`
+    - ➕ `properties`
+    - ( ) `is_requiring_shipping` is renamed to `is_shipping_required`
+  - `shopify__products`:
+    - ➕ `status`
```

Comment:
Same comment from the Source PR. I would prefer we highlight these as breaking changes.